Build error:  not ok /chrono/recurrence_parser - ERROR:test/test-chrono.p/test-chrono.c:1268:test_recurrence_parser: assertion failed: (result != null && result.recurrence != null)

I was trying to package planify for Debian but in building I received this error:

```

4/4 chrono - io.github.alainm23.planify:chrono                         FAIL            0.25s   killed by signal 6 SIGABRT
――――――――――――――――――――――――――――――――――――― ✀  ―――――――――――――――――――――――――――――――――――――
stdout:
TAP version 14
# random seed: R02S041113f1397c88077e2f9a9603628089
1..10
# Start of chrono tests
# Testing: '2024-03-15'
#   Result: 2024-03-15 00:00
# ---
# Testing: '2024-12-25T14:30'
#   Result: 2024-12-25 14:30
# ---
ok 1 /chrono/iso_parser
# Testing: '15/3'
#   Result: 2026-03-15 00:00
# ---
# Testing: '25/12/2024'
#   Result: 2024-12-25 00:00
# ---
# Testing: '1.1.2025'
#   Result: 2025-01-01 00:00
# ---
ok 2 /chrono/slash_parser
# Testing: '14:30'
#   Result: 2026-02-08 14:30
# ---
# Testing: '3pm'
#   Result: 2026-02-08 15:00
# ---
# Testing: '9:30am'
#   Result: 2026-02-08 09:30
# ---
ok 3 /chrono/time_parser
# Testing: 'today'
#   Result: 2026-02-08 00:00
# ---
# Testing: 'tomorrow'
#   Result: 2026-02-09 00:00
# ---
# Testing: 'yesterday'
#   Result: 2026-02-07 00:00
# ---
ok 4 /chrono/casual_date_parser
# Testing: 'morning'
#   Result: 2026-02-08 09:00
# ---
# Testing: 'afternoon'
#   Result: 2026-02-08 14:00
# ---
# Testing: 'midnight'
#   Result: 2026-02-08 00:00
# ---
# Testing: 'noon'
#   Result: 2026-02-08 12:00
# ---
ok 5 /chrono/casual_time_parser
# Testing: 'March 15'
#   Result: 2026-03-15 00:00
# ---
# Testing: '15 March'
#   Result: 2026-03-15 00:00
# ---
# Testing: 'Dec 25, 2024'
#   Result: 2024-12-25 00:00
# ---
# Testing: 'January 2012'
#   Result: 2012-01-01 00:00
# ---
# Testing: 'January'
#   Result: 2026-01-01 00:00
# ---
ok 6 /chrono/month_name_parser
# Testing: 'next week'
#   Result: 2026-02-15 00:00
# ---
# Testing: 'last month'
#   Result: 2026-01-08 00:00
# ---
# Testing: 'this year'
#   Result: 2026-02-08 07:51
# ---
ok 7 /chrono/relative_date_parser
# Testing: 'in 5 minutes'
#   Result: 2026-02-08 07:56
# ---
# Testing: 'in 2 hours'
#   Result: 2026-02-08 09:51
# ---
# Testing: '3 days from now'
#   Result: 2026-02-08 07:51
# ---
ok 8 /chrono/time_expression_parser
# Testing: 'tomorrow at 3pm'
#   Result: 2026-02-09 15:00
# ---
# Testing: 'today at 9:30am'
#   Result: 2026-02-08 09:30
# ---
# Testing: 'March 15 at 14:30'
#   Result: 2026-03-15 14:30
# ---
ok 9 /chrono/datetime_combo_parser
# Testing: 'every day'
not ok /chrono/recurrence_parser - ERROR:test/test-chrono.p/test-chrono.c:1268:test_recurrence_parser: assertion failed: (result != null && result.recurrence != null)
Bail out!
stderr:
**
ERROR:test/test-chrono.p/test-chrono.c:1268:test_recurrence_parser: assertion failed: (result != null && result.recurrence != null)
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――


Summary of Failures:

4/4 chrono - io.github.alainm23.planify:chrono                         FAIL            0.25s   killed by signal 6 SIGABRT

Ok:                3   
Fail:              1   

Full log written to /home/hasan/project/test/planify-4.17.0/obj-x86_64-linux-gnu/meson-logs/testlog.txt
	cd obj-x86_64-linux-gnu && tail -v -n \+0 meson-logs/testlog.txt
==> meson-logs/testlog.txt <==
Log of Meson test suite run on 2026-02-08T07:51:46.692917

Inherited environment: DEB_HOST_MULTIARCH=x86_64-linux-gnu DEB_HOST_GNU_SYSTEM=linux-gnu DEB_BUILD_GNU_TYPE=x86_64-linux-gnu DEB_LDFLAGS_MAINT_APPEND=-Wl,-z,now DEB_TARGET_ARCH_LIBC=gnu DEB_BUILD_ARCH_ENDIAN=little DEB_BUILD_GNU_SYSTEM=linux-gnu DEB_BUILD_ARCH_BITS=64 DEB_BUILD_OPTIONS=parallel=12 DEB_HOST_ARCH=amd64 DEB_TARGET_ARCH_ENDIAN=little SOURCE_DATE_EPOCH=1769444949 DEB_HOST_ARCH_BITS=64 LANG=en_GB.UTF-8 MAKEFLAGS='' DEB_TARGET_ARCH=amd64 DEB_HOST_GNU_CPU=x86_64 DEB_BUILD_ARCH_LIBC=gnu MFLAGS='' DEBEMAIL=modaresisofthard@proton.me DEB_TARGET_ARCH_BITS=64 DEB_BUILD_ARCH_CPU=amd64 DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus DEB_HOST_ARCH_OS=linux DEB_HOST_ARCH_CPU=amd64 MAKELEVEL=1 PATH=/usr/sbin:/usr/bin:/sbin:/bin:/usr/bin/X11 DEB_BUILD_ARCH_ABI=base DEB_BUILD_ARCH=amd64 DEB_HOST_ARCH_LIBC=gnu DEB_TARGET_ARCH_CPU=amd64 DEB_HOST_GNU_TYPE=x86_64-linux-gnu DEB_BUILD_ARCH_OS=linux LOGNAME=hasan DEB_TARGET_MULTIARCH=x86_64-linux-gnu DEB_BUILD_GNU_CPU=x86_64 DEB_TARGET_ARCH_OS=linux DEB_HOST_ARCH_ABI=base DEB_BUILD_MULTIARCH=x86_64-linux-gnu DEB_TARGET_GNU_TYPE=x86_64-linux-gnu DEB_TARGET_ARCH_ABI=base GPG_AGENT_INFO=/run/user/1000/gnupg/S.gpg-agent:0:1 DEB_TARGET_GNU_CPU=x86_64 HOME=/home/hasan/project/test/planify-4.17.0/debian/.debhelper/generated/_source/home DEB_RULES_REQUIRES_ROOT=no DEB_TARGET_GNU_SYSTEM=linux-gnu DEB_HOST_ARCH_ENDIAN=little TERM=xterm-256color DEB_BUILD_MAINT_OPTIONS=hardening=+all DH_INTERNAL_BUILDFLAGS=1 ASFLAGS='' ASFLAGS_FOR_BUILD='' CFLAGS='-g -O2 -Werror=implicit-function-declaration -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection' CFLAGS_FOR_BUILD='-g -O2 -Werror=implicit-function-declaration -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection' CPPFLAGS='-Wdate-time -D_FORTIFY_SOURCE=2' CPPFLAGS_FOR_BUILD='-Wdate-time -D_FORTIFY_SOURCE=2' CXXFLAGS='-g -O2 -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection' CXXFLAGS_FOR_BUILD='-g -O2 -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection' DFLAGS=-frelease DFLAGS_FOR_BUILD=-frelease FCFLAGS='-g -O2 -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-protector-strong -fstack-clash-protection -fcf-protection' FCFLAGS_FOR_BUILD='-g -O2 -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-protector-strong -fstack-clash-protection -fcf-protection' FFLAGS='-g -O2 -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-protector-strong -fstack-clash-protection -fcf-protection' FFLAGS_FOR_BUILD='-g -O2 -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-protector-strong -fstack-clash-protection -fcf-protection' LDFLAGS='-Wl,-z,relro -Wl,-z,now -Wl,-z,now' LDFLAGS_FOR_BUILD='-Wl,-z,relro -Wl,-z,now' OBJCFLAGS='-g -O2 -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection' OBJCFLAGS_FOR_BUILD='-g -O2 -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection' OBJCXXFLAGS='-g -O2 -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection' OBJCXXFLAGS_FOR_BUILD='-g -O2 -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection' GOPATH=/home/hasan/project/test/planify-4.17.0/obj-x86_64-linux-gnu GOCACHE=/home/hasan/project/test/planify-4.17.0/obj-x86_64-linux-gnu/go-build GO111MODULE=off GOPROXY=off GOTOOLCHAIN=local CGO_CFLAGS='-g -O2 -Werror=implicit-function-declaration -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection' CGO_CPPFLAGS='-Wdate-time -D_FORTIFY_SOURCE=2' CGO_CXXFLAGS='-g -O2 -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection' CGO_FFLAGS='-g -O2 -ffile-prefix-map=/home/hasan/project/test/planify-4.17.0=. -fstack-clash-protection -fcf-protection' CGO_LDFLAGS='-Wl,-z,relro -Wl,-z,now -Wl,-z,now' XDG_RUNTIME_DIR=/tmp/dh-xdg-rundir-bPleBU5S DEB_PYTHON_INSTALL_LAYOUT=deb LC_ALL=C.UTF-8 MESON_TESTTHREADS=12 

==================================== 1/4 =====================================
test:         io.github.alainm23.planify:Validate desktop file
start time:   04:21:46
duration:     0.03s
result:       exit status 0
command:      MALLOC_PERTURB_=95 MESON_TEST_ITERATION=1 MSAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1:print_stacktrace=1 ASAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1 UBSAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1:print_stacktrace=1 /usr/bin/desktop-file-validate data/io.github.alainm23.planify.desktop
----------------------------------- stdout -----------------------------------
data/io.github.alainm23.planify.desktop: hint: value "Utility;Office;ProjectManagement;Calendar;" for key "Categories" in group "Desktop Entry" contains more than one main category; application might appear more than once in the application menu
==============================================================================

==================================== 2/4 =====================================
test:         io.github.alainm23.planify:Validate schema file
start time:   04:21:46
duration:     0.02s
result:       exit status 0
command:      MESON_TEST_ITERATION=1 MALLOC_PERTURB_=153 MSAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1:print_stacktrace=1 ASAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1 UBSAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1:print_stacktrace=1 /usr/bin/glib-compile-schemas --strict --dry-run /home/hasan/project/test/planify-4.17.0/data
==============================================================================

==================================== 3/4 =====================================
test:         caldav-integration - io.github.alainm23.planify:caldav-integration
start time:   04:21:46
duration:     0.05s
result:       exit status 0
command:      LD_LIBRARY_PATH=/home/hasan/project/test/planify-4.17.0/obj-x86_64-linux-gnu/core MESON_TEST_ITERATION=1 UBSAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1:print_stacktrace=1 MSAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1:print_stacktrace=1 ASAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1 MALLOC_PERTURB_=237 /home/hasan/project/test/planify-4.17.0/obj-x86_64-linux-gnu/test/test-caldav-integration
----------------------------------- stdout -----------------------------------
TAP version 14
# random seed: R02S12a8936799cdddb4725e33130474343d
1..1
# Start of caldav tests
# MESSAGE: test-caldav-integration.vala:9: Skipping CalDAV login test: missing environment variables.
ok 1 /caldav/login
# End of caldav tests
==============================================================================

==================================== 4/4 =====================================
test:         chrono - io.github.alainm23.planify:chrono
start time:   04:21:46
duration:     0.25s
result:       killed by signal 6 SIGABRT
command:      LD_LIBRARY_PATH=/home/hasan/project/test/planify-4.17.0/obj-x86_64-linux-gnu/core MESON_TEST_ITERATION=1 MSAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1:print_stacktrace=1 MALLOC_PERTURB_=27 ASAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1 UBSAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1:print_stacktrace=1 /home/hasan/project/test/planify-4.17.0/obj-x86_64-linux-gnu/test/test-chrono
----------------------------------- stdout -----------------------------------
TAP version 14
# random seed: R02S041113f1397c88077e2f9a9603628089
1..10
# Start of chrono tests
# Testing: '2024-03-15'
#   Result: 2024-03-15 00:00
# ---
# Testing: '2024-12-25T14:30'
#   Result: 2024-12-25 14:30
# ---
ok 1 /chrono/iso_parser
# Testing: '15/3'
#   Result: 2026-03-15 00:00
# ---
# Testing: '25/12/2024'
#   Result: 2024-12-25 00:00
# ---
# Testing: '1.1.2025'
#   Result: 2025-01-01 00:00
# ---
ok 2 /chrono/slash_parser
# Testing: '14:30'
#   Result: 2026-02-08 14:30
# ---
# Testing: '3pm'
#   Result: 2026-02-08 15:00
# ---
# Testing: '9:30am'
#   Result: 2026-02-08 09:30
# ---
ok 3 /chrono/time_parser
# Testing: 'today'
#   Result: 2026-02-08 00:00
# ---
# Testing: 'tomorrow'
#   Result: 2026-02-09 00:00
# ---
# Testing: 'yesterday'
#   Result: 2026-02-07 00:00
# ---
ok 4 /chrono/casual_date_parser
# Testing: 'morning'
#   Result: 2026-02-08 09:00
# ---
# Testing: 'afternoon'
#   Result: 2026-02-08 14:00
# ---
# Testing: 'midnight'
#   Result: 2026-02-08 00:00
# ---
# Testing: 'noon'
#   Result: 2026-02-08 12:00
# ---
ok 5 /chrono/casual_time_parser
# Testing: 'March 15'
#   Result: 2026-03-15 00:00
# ---
# Testing: '15 March'
#   Result: 2026-03-15 00:00
# ---
# Testing: 'Dec 25, 2024'
#   Result: 2024-12-25 00:00
# ---
# Testing: 'January 2012'
#   Result: 2012-01-01 00:00
# ---
# Testing: 'January'
#   Result: 2026-01-01 00:00
# ---
ok 6 /chrono/month_name_parser
# Testing: 'next week'
#   Result: 2026-02-15 00:00
# ---
# Testing: 'last month'
#   Result: 2026-01-08 00:00
# ---
# Testing: 'this year'
#   Result: 2026-02-08 07:51
# ---
ok 7 /chrono/relative_date_parser
# Testing: 'in 5 minutes'
#   Result: 2026-02-08 07:56
# ---
# Testing: 'in 2 hours'
#   Result: 2026-02-08 09:51
# ---
# Testing: '3 days from now'
#   Result: 2026-02-08 07:51
# ---
ok 8 /chrono/time_expression_parser
# Testing: 'tomorrow at 3pm'
#   Result: 2026-02-09 15:00
# ---
# Testing: 'today at 9:30am'
#   Result: 2026-02-08 09:30
# ---
# Testing: 'March 15 at 14:30'
#   Result: 2026-03-15 14:30
# ---
ok 9 /chrono/datetime_combo_parser
# Testing: 'every day'
not ok /chrono/recurrence_parser - ERROR:test/test-chrono.p/test-chrono.c:1268:test_recurrence_parser: assertion failed: (result != null && result.recurrence != null)
Bail out!
----------------------------------- stderr -----------------------------------
**
ERROR:test/test-chrono.p/test-chrono.c:1268:test_recurrence_parser: assertion failed: (result != null && result.recurrence != null)
==============================================================================


Summary of Failures:

4/4 chrono - io.github.alainm23.planify:chrono                         FAIL            0.25s   killed by signal 6 SIGABRT

Ok:                3   
Fail:              1   
dh_auto_test: error: cd obj-x86_64-linux-gnu && DEB_PYTHON_INSTALL_LAYOUT=deb LC_ALL=C.UTF-8 MESON_TESTTHREADS=12 meson test --verbose returned exit code 1
make: *** [debian/rules:7: binary] Error 25
dpkg-buildpackage: error: debian/rules binary subprocess failed with exit status 2
debuild: fatal error at line 1185:
dpkg-buildpackage -us -uc -ui -i -I failed
gbp:error: 'debuild -i -I' failed: it exited with 29
```

my deps:
```
 libgxml-dev,
 libgio-2.0-dev,
 libgee-0.8-dev,
 libgtk-4-dev,
 libsoup-3.0-dev,
 libsqlite3-dev,
 libadwaita-1-dev,
 libjson-glib-dev,
 libical-dev,
 libsecret-1-dev,
 libspelling-1-dev,
 libspelling-1-dev,
 libgtksourceview-5-dev,
 meson,
 pkgconf,
 valac,
 desktop-file-utils,
 gtk-update-icon-cache,

```

[build_log.txt](https://github.com/user-attachments/files/25157971/build_log.txt)

--- a/test/test-chrono.vala
+++ b/test/test-chrono.vala
@@ -290,47 +290,47 @@ void test_recurrence_parser () {
     var chrono = new Chrono.Chrono ();
     
     print ("Testing: 'every day'\n");
-    var result = chrono.parse ("every day");
+    var result = chrono.parse ("every day", true);
     assert (result != null && result.recurrence != null);
     print ("  Type: %s\n---\n", result.recurrence.recurrence_type.to_string ());
     
     print ("Testing: 'daily'\n");
-    result = chrono.parse ("daily");
+    result = chrono.parse ("daily", true);
     assert (result != null && result.recurrence != null);
     print ("  Type: %s\n---\n", result.recurrence.recurrence_type.to_string ());
     
     print ("Testing: 'every morning'\n");
-    result = chrono.parse ("every morning");
+    result = chrono.parse ("every morning", true);
     assert (result != null && result.recurrence != null);
     print ("  Type: %s, Hour: %d\n---\n", result.recurrence.recurrence_type.to_string (), result.recurrence.hour);
     
     print ("Testing: 'every weekday'\n");
-    result = chrono.parse ("every weekday");
+    result = chrono.parse ("every weekday", true);
     assert (result != null && result.recurrence != null);
     print ("  Type: %s, Days: %d\n---\n", result.recurrence.recurrence_type.to_string (), result.recurrence.days_of_week.size);
     
     print ("Testing: 'every weekend'\n");
-    result = chrono.parse ("every weekend");
+    result = chrono.parse ("every weekend", true);
     assert (result != null && result.recurrence != null);
     print ("  Type: %s\n---\n", result.recurrence.recurrence_type.to_string ());
     
     print ("Testing: 'every 27th'\n");
-    result = chrono.parse ("every 27th");
+    result = chrono.parse ("every 27th", true);
     assert (result != null && result.recurrence != null);
     print ("  Type: %s, Day: %d\n---\n", result.recurrence.recurrence_type.to_string (), result.recurrence.day_of_month);
     
     print ("Testing: 'every last day'\n");
-    result = chrono.parse ("every last day");
+    result = chrono.parse ("every last day", true);
     assert (result != null && result.recurrence != null);
     print ("  Type: %s, Last: %s\n---\n", result.recurrence.recurrence_type.to_string (), result.recurrence.last_day.to_string ());
     
     print ("Testing: 'every jan 27th'\n");
-    result = chrono.parse ("every jan 27th");
+    result = chrono.parse ("every jan 27th", true);
     assert (result != null && result.recurrence != null);
     print ("  Type: %s, Month: %d, Day: %d\n---\n", result.recurrence.recurrence_type.to_string (), result.recurrence.month_of_year, result.recurrence.day_of_month);
     
     print ("Testing: 'every hour'\n");
-    result = chrono.parse ("every hour");
+    result = chrono.parse ("every hour", true);
     assert (result != null && result.recurrence != null);
     print ("  Type: %s\n---\n", result.recurrence.recurrence_type.to_string ());
 }

